modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -116,6 +116,7 @@
     protected Map<String, String> typeAliases = null;
 
     protected String ignoreFilePathOverride;
+    private boolean notNullJacksonAnnotation;
```

Comment:
this is specific to a generator, not to the whole set, DefaultCodegen must be left unchanged, you would code this into the Java generator